samples/display-map-from-mobile-map-package/src/main/java/com/esri/arcgismaps/sample/displaymapfrommobilemappackage/MainActivity.kt

```diff
@@ -17,57 +17,38 @@
 package com.esri.arcgismaps.sample.displaymapfrommobilemappackage
 
 import android.os.Bundle
-import android.util.Log
-import android.view.View
+import androidx.activity.compose.setContent
 import androidx.appcompat.app.AppCompatActivity
-import androidx.databinding.DataBindingUtil
-import androidx.lifecycle.lifecycleScope
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
 import com.arcgismaps.ApiKey
 import com.arcgismaps.ArcGISEnvironment
-import com.arcgismaps.mapping.MobileMapPackage
-import com.esri.arcgismaps.sample.displaymapfrommobilemappackage.databinding.DisplayMapFromMobileMapPackageActivityMainBinding
-import com.google.android.material.snackbar.Snackbar
-import kotlinx.coroutines.launch
-import java.io.File
+import com.esri.arcgismaps.sample.displaymapfrommobilemappackage.screens.MainScreen
+import com.esri.arcgismaps.sample.sampleslib.theme.SampleAppTheme
 
 class MainActivity : AppCompatActivity() {
```

Comment:
Change this so that the title is only displayed by MainScreen

![Screenshot 2024-11-19 at 12 06 10 PM](https://github.com/user-attachments/assets/86299fdd-2314-4c71-91f8-620232909e21)

```suggestion
class MainActivity : ComponentActivity() {
```